src/main/java/me/ryanhamshire/GriefPrevention/CreateClaimResult.java

```diff
@@ -21,9 +21,15 @@
 public class CreateClaimResult
 {
     //whether or not the creation succeeded (it would fail if the new claim overlapped another existing claim)
-    public boolean succeeded;
+    public Value value;
 
     //when succeeded, this is a reference to the new claim
     //when failed, this is a reference to the pre-existing, conflicting claim
     public Claim claim;
+
+    public enum Value {
+        SUCCEEDED,
+        FAILED,
+        CANCELLED
```

Comment:
Where is this used...?

Reply:
When cancelling the ClaimModifiedEvent, we don’t want GP to act on a cancelled event.